src/fides/api/ctl/routes/admin.py

```diff
@@ -23,7 +23,7 @@ async def db_action(action: DBActions) -> Dict:
     """
     action_text = "initialized"
     if action == DBActions.reset:
-        database.reset_db(CONFIG.database.sync_database_uri)
+        database.reset_db(str(CONFIG.database.sync_database_uri))
         action_text = DBActions.reset
-    await database.configure_db(CONFIG.database.sync_database_uri)
+    await database.configure_db(str(CONFIG.database.sync_database_uri))
```

Comment:
It goes through several layers before it gets here, but once it reaches `set_main_option` this method from alembic requires a string. Since it eventually has to be converted I did it here because it required the least changes.

Reply:
Understandable. Maybe the most robust solution is to change the type assertion on the config option to be a `str`, and implement a validator to ensure the string contains a valid `PostgresDsn`?

Reply:
This could work. I'll give it a try and see if everything happy.

Reply:
I didn't see any errors after converting it to a string in the config so I implemented it.